adot-testbed/app/src/test/java/software/amazon/adot/testbed/LogsTests.java

```diff
@@ -160,6 +271,13 @@ void validateLogs(String testLogStreamName, String logFilePath) throws Exception
                     .filter(Objects::nonNull)
                     .collect(Collectors.toSet());
 
+                //Print
+                System.out.println("Actual Logs - Log lines From Cloudwatch");
+                messageToValidate.forEach(System.out::println);
+
+                System.out.println("Expected logs- Log lines from log file");
+                lines.forEach(System.out::println);
```

Comment:
Let's be more thoughtful about what we are printing to std out. We should only need to print if there is a failure. And in the case of the failure it should be descriptive to tell us what was expected and what was actually given.

Reply:
Sure, I have added it for easy lookout on whats' validated. if we should only need to print during failure then the assertions will log the actual/expected when in failure, removed the print statements